src/main/scss/base/_style.scss

```
@@ -290,7 +290,7 @@ pre.console {
   display: none; /* hidden until loaded */
   padding: 1rem;
   margin: 1rem 0;
-  word-break: word-break;
+  word-break: break-word;
```

```suggestion
  word-break: word-break;
```

---

It seems that [mozilla instead suggests](https://developer.mozilla.org/en-US/docs/Web/CSS/word-break#break-word)
```suggestion
  overflow-wrap: anywhere;
  word-break: normal;
```
but I'll have to test that

---

Thanks @lewisbirks!